src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/IInvitationRepository.cs

```diff
@@ -8,5 +8,6 @@ public interface IInvitationRepository : IRepository<Invitation>
         void RemoveCommPkg(CommPkg commPkg);
         void RemoveMcPkg(McPkg mcPkg);
         void RemoveParticipant(Participant participant);
+        void RemoveAttachment(Attachment attachment);
```

Comment:
Missing test for RemoveAttachment in InvitationRepositoryTests.
When investigating InvitationRepositoryTests, I see a lot of test missing there + a lot of existing tests not belonging there.